express/index.html

```diff
@@ -223,7 +225,8 @@ <h2 class="title text-dark">Nosso time</h2>
 							<div><img src="./assets/img/site-img/valesca.png" alt="Valesca Fortunato"
 									class="team-profile-img"></div>
 							<h4>Valesca Fortunato</h4>
-							<p style="padding:10px 5px;">Valesca trabalha com Infraestrutura de TI há mais de 18 anos, é formada em Gestão de
+							<p style="padding:10px 5px;">Valesca trabalha com Infraestrutura de TI há mais de 18 anos, é
```

Comment:
Can you put this css code `padding:10px 5px;` inside file `assets/css/style.css`?

Reply:
Done.